submit-queue/whitelist.txt

```diff
@@ -1,41 +1,6 @@
-brendandburns
-thockin
-mikedanese
-a-robinson
-saad-ali
-lavalamp
```

Comment:
better explanation, the point of the PR was to pull people with commit access from github and consider them 'whitelisted'  I'm not trying to maintain that list in the whitelist.txt.  We just pull it when the submit-queue launches. So now we only have to maintain in this file those people without commit access that we want to be able to use the submit queue without the extra label.